src/configure/helper/commonHelper.ts

```diff
@@ -6,9 +6,22 @@ export async function sleepForMilliSeconds(timeInMs: number): Promise<void> {
     });
 }
 
+export function generateDevOpsOrganizationName(userName: string, repositoryName: string): string {
+    let repoParts = repositoryName.split("/");
+    let repositoryNameSuffix = repoParts[repoParts.length-1];
+    repositoryNameSuffix = repositoryName.trim();
+    // Generate random 2 digit number
+    let uniqueId = Math.floor((Math.random()*90) + 10);
```

Comment:
Should we be generating two random characters? If we use number people might question why does it now follow a sequence.
And do not use `-` between repo name and uniqueId

Reply:
As discussed offline, we will not add guid in generated name, input will be shown if name already exists.